support-frontend/assets/pages/supporter-plus-landing/twoStepPages/checkoutScaffold.tsx

```diff
@@ -167,9 +167,26 @@ export function SupporterPlusCheckoutScaffold({
 
 	useEffect(() => {
 		if (paymentComplete) {
-			navigateWithPageView(navigate, thankYouRoute, abParticipations, {
-				replace: true,
-			});
+			const urlParams = new URLSearchParams(window.location.search);
```

Comment:
I've moved this here as supporterPlusRouter is instantiated when you hit the `/contribute` page meaning it will use those parameters rather than from the /checkout` page.